reactor-netty-http/src/main/java/reactor/netty/http/server/logging/AccessLogFactory.java

```diff
@@ -16,18 +16,57 @@
 package reactor.netty.http.server.logging;
 
 import java.util.function.Function;
+import java.util.function.Predicate;
 
-/**
+ /**
  * An interface to declare more concisely a {@link Function} that apply an {@link AccessLog} by an
  * {@link AccessLogArgProvider}.
  * <p>
- * Can be used in {@link reactor.netty.http.server.HttpServer#accessLog(boolean, AccessLogFactory) accessLog} method
- * for example.
+ * Can be used in {@link reactor.netty.http.server.HttpServer#accessLog(boolean, AccessLogFactory) accessLog} method for example.
  *
  * @author Simon Baslé
  * @author Audrey Neveu
  * @since 1.0.3
  */
 public interface AccessLogFactory extends Function<AccessLogArgProvider, AccessLog> {
 
-}
\ No newline at end of file
+	/**
+	 * Helper method to create an access log factory that selectively enables access logs.
+	 * <p>
+	 * Any request (represented as an {@link AccessLogArgProvider}) that doesn't match the
+	 * provided {@link Predicate} is excluded from the access log. Other requests are logged
+	 * using the default format.
+	 *
+	 * @param predicate the filter that returns {@code true} if the request should be logged, {@code false} otherwise
+	 * @return an access log factory {@link Function} to be used in
+	 * {@link reactor.netty.http.server.HttpServer#accessLog(boolean, AccessLogFactory)}
+	 * @since 1.0.3
+	 */
+	static AccessLogFactory createFilter(Predicate<AccessLogArgProvider> predicate) {
+		return input -> predicate.test(input) ? BaseAccessLogHandler.DEFAULT_ACCESS_LOG.apply(input) : null;
+	}
+
+	/**
```

Comment:
there seems to be something wrong with how your IDE is configured: there's an extra space before each comment line in this block

Reply:
I don't really know why is that as I don't see anything special in my configuration 😞

Reply:
see above, this block was actually fine.

there is however now an extra leading space in some of the pseudo-blank lines below
![image](https://user-images.githubusercontent.com/6986166/103680589-ace78c80-4f86-11eb-84f9-454dd8dbf658.png)